Sofa/Component/Topology/Container/Constant/src/sofa/component/topology/container/constant/MeshTopology.cpp

```diff
@@ -535,17 +535,21 @@ void MeshTopology::init()
     BaseMeshTopology::init();
 
     const auto& hexahedra = seqHexahedra.getValue();
```

Comment:
Shouldn't it be better to use an accessor there ?

Reply:
I don't know. Is there a difference between `getValue` and `ReadAccessor`?